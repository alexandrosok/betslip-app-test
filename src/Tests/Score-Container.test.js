import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import {shallowToJson} from 'enzyme-to-json';
import ScoreContainer from "../components/Score-Container/score-container";
configure({adapter: new Adapter()})

describe('ScoreContainer', () => {
    const component = shallow(<ScoreContainer homeScore="mockSport" awayScore="mockEventName"/>);

    it('should render correctly', () => {
        expect(shallowToJson(component)).toMatchSnapshot();
    });
});

describe('ScoreContainer containers', () => {
    const component = shallow(<ScoreContainer homeScore="mockSport" awayScore="mockEventName"/>);

    it('should have a div with class div.score-container', () => {
        expect(component.find('div.score-container').length).toEqual(1)
    })
});

