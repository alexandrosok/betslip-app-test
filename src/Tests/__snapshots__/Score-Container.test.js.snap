// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScoreContainer should render correctly 1`] = `
<div
  className="score-container"
>
  mockSport
   - 
  mockEventName
</div>
`;
